src/lib/viewers/controls/annotations/annotations-button-region-tooltips/AnnotationsButtonRegionTooltips.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext, PropsWithChildren } from 'react';
```

Comment:
Can we remain consistent with the existing codebase conventions? We typically use the `React` namespace for these helpers.

Reply:
Sure. I have always found named imports to be a cleaner pattern, but you're right that being consistent is even more important.

Reply:
Done. Resolving this.